src/components/category-template.js

```diff
@@ -49,11 +49,16 @@ export default function CategoryTemplate(props) {
   const hasMorePosts = morePosts.length > 0;
   const isFirstPage = pageInfo.currentPage === 1;
 
+  const parentCategorySlug = wpParent?.node?.name
+    ? `${wpParent.node.name} > ${name}`
+    : '';
```

Comment:
the ternary doesn't seem to output the correct value for top-level categories?
For this url https://deploy-preview-59--apollographql-blog.netlify.app/platform/1 I get:
 ` | 1 - Apollo GraphQL Blog` page title instead of `Platform | 1 - Apollo GraphQL Blog`
<img width="235" alt="Screenshot 2022-06-15 at 17 24 30" src="https://user-images.githubusercontent.com/1368445/173865395-9dba16fc-8b1e-497f-8db4-31445c023b88.png">

The last part of the ternary should probably be the `name` value if no parent category?

Reply:
Ah nice catch, my bad! This is fixed up.